### ISSUE TITLE
Upgrade jquery 1.6.4

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmithdemos/html5rocks#554]
```
https://ajax.googleapis.com/ajax/libs/jquery/1.6.1/jquery.min.js
```

Original issue reported on code.google.com by `ericbide...@html5rocks.com` on 28 Jun 2011 at 1:55

**Comment by GoogleCodeExporter** [comment]
Original comment by `ericbide...@html5rocks.com` on 18 Aug 2011 at 6:04
- Added labels: **Milestone-7.1**

**Comment by GoogleCodeExporter** [comment]
```
Removed extra jquery includes now that we have a single include on the base:
http://code.google.com/p/html5rocks/source/detail?r=527c52889e31
```

Original comment by `ericbide...@html5rocks.com` on 14 Oct 2011 at 10:55
- Changed title: **Upgrade jquery 1.6.4**
- Changed state: **Fixed**
- Added labels: **Priority-P1**, **Milestone-Q42011-1**, **redesign**
- Removed labels: **Priority-P2**, **Milestone-7.1**
